svm/src/transaction_processor.rs

```diff
@@ -284,19 +298,20 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
                         &programs_loaded_for_tx_batch.borrow(),
                     );
 
-                    if let TransactionExecutionResult::Executed {
-                        details,
-                        programs_modified_by_tx,
-                    } = &result
-                    {
-                        // Update batch specific cache of the loaded programs with the modifications
-                        // made by the transaction, if it executed successfully.
-                        if details.status.is_ok() {
-                            programs_loaded_for_tx_batch
-                                .borrow_mut()
-                                .merge(programs_modified_by_tx);
-                        }
-                    }
+                    // Intentionally omitted for brevity:
```

Comment:
I'm not following these _brevity omissions_ -- doesn't this change the semantics of `load_and_execute_sanitized_transactions`?